translations/en-us.yml

```diff
@@ -1242,6 +1242,8 @@ preprints:
             no-conflict-of-interest: 'Author asserted no Conflict of Interest.'
             public-data: 'Public Data'
             public-preregistration: 'Public Preregistration'
+            supplement-title: 'OSF Project'
```

Comment:
Another question for product, but do we want  "OSF Project" here, or something a little more generic like "Supplemental Materials"?